app/services/collators/capital_collator.rb

```diff
@@ -45,9 +45,10 @@ def perform_assessments
         total_mortgage_allowance: property_maximum_mortgage_allowance_threshold,
         total_property: property_value,
         pensioner_capital_disregard: @pensioner_capital_disregard,
+        pensioner_disregard_applied: [@pensioner_capital_disregard, total_capital].min,
         subject_matter_of_dispute_disregard: subject_matter_of_dispute_disregard + property_smod,
         total_capital:,
-        assessed_capital:,
+        assessed_capital: [assessed_capital, 0].max,
```

Comment:
This feels to me like it's potentially a breaking change. Have we checked with Apply that in situations where pensioner clients have less than 100k in assets, they're happy for assessed capital to stop being a negative number?

Reply:
I cant find anywhere in the Apply codebase that the `assessed_capital` value is being used, it certainly is not being used to display on the results page and that would be the only likely candidate for displaying this information.

Reply:
Personally I was more worried that this might be a breaking change for CCQ. Does it rely on assessed_capital being negative for some of its calculations?

Reply:
Thanks @willc-work I think you're right, they don't use it. And yup, @starswan you're right, we don't rely on it being negative - for the applicant we ignore it (but would like to start using it), and for the partner we compensate for its possible negativity explicitly. I had actually asked for a never-negative version of this as part of EL-767, but no complaints from our end if it gets added in in an earlier ticket!

Reply:
Ugh, we were _so close_ to catching this one in code review!